tests/test_serving.py

```diff
@@ -171,12 +171,6 @@ def test_chunked_encoding(monkeypatch, dev_server, send_length):
             ),
         }
     )
-    headers = {"content-type": f"multipart/form-data; boundary={boundary}"}
-
-    if send_length:
-        headers["transfer-encoding"] = "chunked"
-        headers["content-length"] = str(length)
-
```

Comment:
Looks like this was intended to be removed after the changes to use `putheader` (below) were introduced.